tests/test_strace.py

```diff
@@ -6,12 +6,16 @@
 import tempfile
 import unittest
 import signal
+import shutil
+import platform
 
 STRACE = os.path.normpath(
     os.path.join(os.path.dirname(__file__), '..', 'strace.py'))
 
 AARCH64 = (getattr(os.uname(), 'machine', None) == 'aarch64')
 
+UNSUPPORTED = platform.system() not in ('Linux',)
```

Comment:
python-ptrace supports FreeBSD, no? Why only running strace.py tests on Linux?

Reply:
Yes, but the github actions do not support FreeBSD, and I have no way of testing this. I took a conservative approach, avoiding to break tests untested systems.

Reply:
If the test is skipped and someone runs the tests manually, it's too easy to miss that the whole test is skipped. I prefer to continue to run the test, and only skip a specific test if there is no way to fix it.

Reply:
But isn't it what I do here ? I only skip a single test: `@unittest.skipIf(UNTESTED, 'Untested system/OS')` ...